### DISCUSSION TITLE
Need some CSS

### DISCUSSION THREAD

**Discussion by khabib227** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#75] [Q&A] [answered]
What css I need to set the suggestions font size? To hide the 3 dots (...) and to customise the scrollbar, all in suggestion list like:

{scrollbar-color: #0A84FF #000 !important;
  scrollbar-width: thin !important;}

What id or class to suggestion scrollbar?

What css I use to remove the delay from folders, so, to show the folder content immediately? 

Thanks!

**Comment by YoshifumiFuyuno** [comment]
Hi.

The display of input element candidates is controlled by the browser and probably cannot be changed.
possibly it can be changed, but I don't know how, so I can't help you.

"Popup/icons behaviour" -> "Wait time to open folder" to 200.
"Popup/icons behaviour" -> "Popup animation duration" to 0.
Or, if you want the folder to appear instantly with the animation remaining.
[Custom CSS](https://github.com/YoshifumiFuyuno/Search-from-Popup-or-ContextMenu/wiki/Custom-CSS) -> "Subfolder display speed: 0s"

Best regards

**Comment by khabib227** [comment]
Thanks!  The "Wait time to open folder" don't support values less than 200, and I added "Subfolder display speed: 0s" to custom css but still a delay to open folders with mouse over. Has some css to remove the delay completely?

**Comment by YoshifumiFuyuno** [comment]
🤔I'm not sure because there are no delays in my environment.
You're talking about opening the popup folder, right?
How much delay is there specifically?

If you're talking about context menus, context menus are managed by the browser.

**Comment by khabib227** [comment]
I'm talking when I put the cursor over an engines folder, not about context menus, here I'm using the last Win 10, I have more or less 250-500ms of delay to open the folder. The minimum "Wait time to open folder" is 200ms, so I think I have 200ms of delay exactly.

**Comment by YoshifumiFuyuno** [comment, marked as answer]
It can be reduced to less than 200 in this way. (However, something may go wrong.)
If it's hard to understand, please ask me.

https://github.com/YoshifumiFuyuno/Search-from-Popup-or-ContextMenu/assets/9511854/fb58debd-6f51-46ad-a5bd-acfb82feb8a1

**Comment by khabib227** [comment]
Thanks, is it! Now I have zero delay to open and close folders! If possible remove this minimum 200 in some next release.
